packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js

```diff
@@ -134,6 +134,19 @@ const CardanoTestnetExplorers: Array<$ReadOnly<ExplorerRow>> = [
   },
 ];
 
+const CardanoPreprodTestnetExplorers: Array<$ReadOnly<ExplorerRow>> = [
+  {
+    ExplorerId: 4_00,
```

Comment:
The duplication of the `ExplorerId` to be [400](https://github.com/Emurgo/yoroi-frontend/blob/78761b871aee55fafbe6ab447813932b1b083e1f/packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js#L126) is causing the extension to crash when restoring a wallet on the `Legacy` testnet. The Id must be unique. It took me a while to debug this :)

Reply:
oh heck, good catch! Thank you, @AhmedIbrahim336, fixing it now!

Reply:
Updated now, and replaced with cardanoscan, plz check again

Reply:
Sounds good, thanks @vsubhuman